Clinical-Genomics/MUTANT
# Structural cleanup

### The purpose of the code changes are as follows:
- Restructured the folder structure and script names
- Hasta update script explicitly added
- Renamed commands to be more intuitive
- Cleaned up self-test
- Trimmed script invokations from unnecessary variables
- Pangolin consensus file is now generated
- Added single function to produce all reports
- Subdivided  single script into reports and delivery
- Rewrote artic_report script to be part of the report module
- Generated images are now chmodded to allow all access
- Blacked everything

**How to prepare for test**:
- `cd MUTANT`
- `export PATH=$PATH:MUTANT_DIR`
- `source activate CONDA_ENV`
- `pip install -e .`

### How to test:
- cd to the MUTANT directory and run `mutant analyse sarscov2 tests/testdata/fasta_files --profiles local,singularity --config_artic mutant/config/local/artic.json --config_mutant mutant/config/local/mutant.json`
- Check that no new commands collide with https://github.com/Clinical-Genomics/atlas/blob/6f7a6b4ba3b67a098c9e8ccb9a3fd46881fe82b9/docs/infrastructure/production/workflows/sars-cov-2.md
- Install on stage and run against frankhusky. All files mentioned in trailblazer file needs to be populated with reasonable data.

### Expected outcome:
- [x] Produced files contain expected values

### Review:
- [x] Code reviewed by @talnor @henningonsbring 

This [version](https://semver.org/) is a:
